testsuite/scripts/src/test/java/org/wildfly/scripts/test/CliScriptTestCase.java

```diff
@@ -42,7 +44,10 @@ public CliScriptTestCase() {
     @Override
     void testScript(final ScriptProcess script) throws InterruptedException, TimeoutException, IOException {
         // Read an attribute
-        script.start(MAVEN_JAVA_OPTS, "--commands=embed-server,:read-attribute(name=server-state),exit");
+        Map<String, String> env = new LinkedHashMap<>(MAVEN_JAVA_OPTS);
+        // https://issues.redhat.com/browse/WFCORE-5241 Test with JAVA_OPTS parameter including whitespace
+        env.put("JAVA_OPTS", "-Dtestparameter=\"something with space\"");
```

Comment:
@soul2zimate I think you should check if JAVA_OPTS already exists and append to it. You can look at: https://github.com/wildfly/wildfly-core/blob/master/testsuite/scripts/src/test/java/org/wildfly/scripts/test/ScriptTestCase.java#L86

Reply:
hmmm, that didn't show at local run. I reckon teamcity have something more configured than my local one.
I pushed an update to check any existing JAVA_OPTS first.